<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkView</class>
 <widget class="QWidget" name="NetworkView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wireless Monitor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_main">
   <item row="0" column="0" colspan="10">
    <widget class="QWidget" name="widget_contain" native="true">
     <layout class="QGridLayout" name="gridLayout_setting">
      <item row="0" column="7">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Background Image / Layout</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="pb_openjob">
           <property name="text">
            <string>Open Job</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cb_floor">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>ZigBee Device</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_0">
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="ch_comboBox"/>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>Scan</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="cbZS100">
           <property name="text">
            <string>ZS-100</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Scroll</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_1">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Auto</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Snap Shot</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnPrint">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>PCAP Save</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enable</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="saveFileName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>File name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnDir">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>./</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>PCAP Load</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="10">
    <widget class="QTabWidget" name="tabWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnPrint</sender>
   <signal>clicked()</signal>
   <receiver>NetworkView</receiver>
   <slot>onPrint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>903</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDir</sender>
   <signal>clicked()</signal>
   <receiver>NetworkView</receiver>
   <slot>onSaveDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onScan(bool)</slot>
  <slot>onzbScan(bool)</slot>
  <slot>openFile()</slot>
  <slot>onPrint()</slot>
  <slot>onSaveDir()</slot>
 </slots>
</ui>
